src/Snapshot/SnapshotExpired.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Patchlevel\EventSourcing\Snapshot;
+
+use function sprintf;
+
+final class SnapshotExpired extends SnapshotException
```

Comment:
@DanielBadura maybe you have a better name for it. I think "expired" doesn't quite fit.

Reply:
Maybe call it for what it really stand for: SnapshotVersionInvalid